backend/pkg/version/version.go

```diff
@@ -0,0 +1,60 @@
+package version
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"runtime"
+	"runtime/debug"
+	"text/tabwriter"
+)
+
+var (
+	version string = "v0.0.0-dev"
+)
+
+type Version struct {
+	Version      string
+	GoVersion    string
+	GitCommit    string `json:",omitempty"`
+	RevisionTime string `json:",omitempty"`
+	OS           string
+	Arch         string
+}
+
+func Get() Version {
+	v := Version{
+		Version:   version,
+		GoVersion: runtime.Version(),
+		OS:        runtime.GOOS,
+		Arch:      runtime.GOARCH,
+	}
+	if bldInfo, ok := debug.ReadBuildInfo(); ok {
+		for _, setting := range bldInfo.Settings {
+			switch setting.Key {
+			case "vcs.revision":
+				v.GitCommit = setting.Value
+			case "vcs.time":
+				v.RevisionTime = setting.Value
```

Comment:
great use of the vcs which is populated by default since some time now. I just love this patch.